chapters/hpmor-chapter-022.tex

```diff
@@ -1,5 +1,12 @@
 \chapter{The Scientific Method}
 
+\begin{chapterOpeningAuthorNote}
+The key to strategy is not to choose a path to J. K. Rowling, but to choose so that all paths lead to a J. K. Rowling.
```

Comment:
Have we checked for consistency? Should in any case be "J.~K.~Rowling" (so we don't get line breaks between an initial and the surname).

Reply:
good point, fixed in all files

Reply:
added the check to https://github.com/rrthomas/hpmor/pull/69 as well